src/storage/test/DeleteTagsTest.cpp

```diff
@@ -164,7 +164,8 @@ TEST(DeleteTagsTest, SimpleTest) {
     auto resp = std::move(fut).get();
 
     ASSERT_EQ(0, (*resp.result_ref()).failed_parts.size());
-    ASSERT_EQ(0, resp.props_ref()->size());
```

Comment:
Because I need modify mock data and add vertex key into it. Before that, GetProp can't get any data if tag has been deleted.But now it can get a vertex without any tag, and resultDataSet is not empty.